### PR TITLE
53 docs init pattern

### PR DESCRIPTION
**Pull request by jalew188**
For a class, move init params into __init__ method,  instead of the class definition.

### CODE REVIEW

File: alpharaw/wrappers/alphatims_wrapper.py [hunk 1]
```diff
@@ -45,25 +45,26 @@ def import_raw(self, burker_d_folder: str):
...
+
+        dda : bool
+            If DDA, precursor indices will be equal to scan numbers.
+            If not DDA (i.e. DIA), precursor indices will be equal to the
+            scan number within a DIA cycle.
+
+        slice_as_dataframe : bool
+            If True, slicing returns a pd.DataFrame by default.
+            If False, slicing provides a np.int64[:] with raw indices.
+            This value can also be modified after creation.
```
Comment (mschwoer) [inline alpharaw/wrappers/alphatims_wrapper.py:65]: "This value can also be modified after creation." -> what does that mean?

File: alpharaw/wrappers/alphatims_wrapper.py [hunk 2]
```diff
@@ -45,25 +45,26 @@ def import_raw(self, burker_d_folder: str):
...
+        ----------
+        msdata : MSData_Base
+            The AlphaRaw data object.
+
+        dda : bool
+            If DDA, precursor indices will be equal to scan numbers.
+            If not DDA (i.e. DIA), precursor indices will be equal to the
+            scan number within a DIA cycle.
+
+        slice_as_dataframe : bool
```
Comment (mschwoer) [inline alpharaw/wrappers/alphatims_wrapper.py:62]: is this parameter even used somewhere?
(same for `_use_calibrated_mz_values_as_default`, `thermo_raw_file_name`, `bruker_d_folder_name`)

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 1 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:65]: This module will be deprecated, as alphadia has its own implementation already, so I did not work on it any more.

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 2 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: See above.

File: alpharaw/thermo.py [hunk 3]
```diff
@@ -82,6 +66,23 @@ def __init__(
         auxiliary_items: list = [],
         **kwargs,
     ):
+        """
+        Parameters
+        ----------
+        centroided : bool, optional
+            If peaks will be centroided after loading. By defaults True.
```
[the remark this answers is not in the document]
Comment (jalew188) [reply alpharaw/thermo.py:73]: Yes, will do.

[the remark this answers is not in the document]
Comment (jalew188) [reply alpharaw/thermo.py:73]: Done

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 2 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: wont FIX

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: See https://github.com/MannLabs/alphadia/blob/main/alphadia/data/alpharaw.py#L251

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 1 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:65]: See https://github.com/MannLabs/alphadia/blob/main/alphadia/data/alpharaw.py#L251

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:65]: > wont FIX:)

Lets MB keep it ..

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 2 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: > wont FIX:)

Lets MB keep it ..

Comment (mschwoer) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: I see .. I would vote for removing it or moving the alphaDIA implmentation here in order to have less dead code

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: Keep it in AlphaDIA as alphaDIA is better at handling numba jit classes or functions

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: But I will still keep it in case alphaViz needs it

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:62]: `slice_as_dataframe` is required by the super class `AlphaTims`, it must be set here to enable alphatims functionalities.

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 1 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:65]: This is required by the super class AlphaTims, it must be set here to enable alphatims functionalities.

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:65]: I will move this to the attribute definition. It means this attr can be modified on-the-fly.

Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py]: Moved this to attribute definition.

File: alpharaw/wrappers/alphatims_wrapper.py (hunk 2 above)
Comment (jalew188) [reply alpharaw/wrappers/alphatims_wrapper.py:70]: Done

**Review by GeorgWa — APPROVED** [review]
LGTM

**Pull request merged**